api/include/opentelemetry/context/context.h

```diff
@@ -16,6 +16,8 @@ class Context
 {
 
 public:
+  Context() = default;
+
```

Comment:
Can you give some context on why the constructor is moving to public visibility?  If I recall correctly it was intentionally private.

Reply:
My apologies, I was testing something and forgot to move it back.